README.md

```diff
@@ -1,44 +1,68 @@
 # pwned-password-check-net
-Simple .net core command line utility for checking your pwned passwords against Troy Hunt's Have I Been Pwned? service.
+Simple .net core command line utility for checking your pwned accounts against Troy Hunt's Have I Been Pwned? service. Export accounts into CSV file and run pwned-password-check-net.dll.
 
+> dotnet pwned-password-check-net.dll -p accounts.csv -l KeePass
 
-# Usage
-## Build from source code
+# Install
+## Windows
+### Build from source code
 Clone this repository and compile with dotnet tooling. (or visual studio code)
-## Download
+### Download
 Download from AppVeyor continuous integration.
-[Latest build](https://ci.appveyor.com/api/projects/hopp/pwned-password-check-net/artifacts/pwned-password-check-net.zip?bramch=master)
-
-## Export passwords
-Export your passwords to csv file, with structure
-
-| Account | LoginName | Password | WebSite |
-| ---     | --- | ---| --- |
-| My favourite service|your_account@your_domain.com|password|www.myfavouriteservice.nowhere
-This is default export from KeePass.
-## Run
-There are two parameters
-|Paremeter|Long Name|Description|
+[Latest build](https://ci.appveyor.com/api/projects/hopp/pwned-password-check-net/artifacts/pwned-password-check-net.zip?branch=master)
+
+## Linux
+    sudo snap install dotnet-sdk --classic
+    mkdir ./pwned && cd ./pwned
+    wget https://ci.appveyor.com/api/projects/hopp/pwned-password-check-net/artifacts/pwned-password-check-net.zip?branch=master -O master.zip
+    unzip master.zip
+    chmod 644 *
+
+# Prepare accounts
+## Linux
+    cat << EOF > accounts.csv
+    Account,LoginName,Password,WebSite,Comments
+    my_gaccount,firstname.lastname@gmail.com,-,gmail.com,email provider
+    my_yaccount,firstname.lastname@yahoo.com,-,yahoo.com,email provider
+    EOF
+
+## Manual
+Export your passwords to csv file with following structure:
+
+| Account | LoginName | Password | WebSite | Comments |
+| --- | --- | ---| --- | --- |
+| My favourite service|your_account@your_domain.com|password|www.myfavouriteservice.nowhere|comment|
+
+Note: This is default export from KeePass.
+
+# Run
+
+## Windows
+    dotnet pwned-password-check-net.dll -p path_to_csv -l KeePass
+
+## Linux
+    dotnet-sdk.dotnet pwned-password-check-net.dll -p path_to_csv -l KeePass
```

Comment:
In case of .net core installed on the system (not in snap) the path should be the same as on Windows